backend/src/interface/controller/bookController.ts

```diff
@@ -118,6 +117,19 @@ export default class BookController {
       return;
     }
 
-    context.response.status(200).json({ ...books });
+    context.response.status(200).json({ ...book });
```

Comment:
Igual aquí debe ser "books"
```suggestion
    context.response.status(200).json({ ...books });
```